go.mod

```diff
@@ -1,3 +1,3 @@
 module nhooyr.io/websocket
 
-go 1.19
+go 1.22.5
```

Comment:
We want to retain go 1.19 compatibility for now, bumping to 1.22 is too big a jump anyway as 1.20 would be the next one.